.azure-ci/docker_scripts.sh

```diff
@@ -41,6 +41,7 @@ pip install wheel
 pip install -e ".[dev]"
 
 # Test dev install with pytest
+CI=1 # makes hypothesis print falsifying examples
```

Comment:
Interesting, I had no idea!  Could you share a link with an explanation of this?

Reply:
To be more specific, it prints a blob which can be used with `@reproduce_failure`, see [the documentation of hypothesis](https://hypothesis.readthedocs.io/en/latest/settings.html?highlight=reproduce#hypothesis.settings.print_blob). Then, you can run this test with a debugger.

Reply:
Thanks!